plugins/wmap.rb

```diff
@@ -147,23 +147,28 @@ def cmd_wmap_sites(*args)
         when '-s'
           u = args.shift
           l = args.shift
-          s = args.shift
+          o = args.shift
 
-          if not u
-            return
-          end
+          return unless u
 
           if l == nil or l.empty?
             l = 200
-            s = true
+            o = 'true'
           else
-            l = l.to_i
-            s = false
+            # Add check if unicode parameters is the second one
+            if l == 'true' or l == 'false'
+              o = l
+              l = 200
+            else
+              l = l.to_i
+            end
           end
 
+          o = (o == 'true')
```

Comment:
That's right, an other option is this :
```
          o = args.shift
          o = (o == 'true')

          return unless u

          if l == nil or l.empty?
            l = 200
            o = true
          else
            # Add check if unicode parameters is the second one
            if l == 'true' || l == 'false'
              o = (l == 'true')
              l = 200
            else
              l = l.to_i
            end
          end
```
But it's involves two resolutions, not necessarily prettier…